_includes/work-at-triggerise.html

```diff
@@ -1,40 +1,29 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-</head>
-  <body>
-    <div class="uk-child-width-1-2@m uk-grid-small uk-grid-match uk-grid">
-      <div class="uk-child-width-auto@m uk-text-left uk-margin-auto">
-        <div class="uk-blockquote lightBlue uk-card-default uk-card-body">
-              <blockquote class="blockquote blockquote--quoted">
-                <p class="blockquote__text">
-                    We come from different cultures and backgrounds; 
-                    what unites us is our open-mindedness and drive for impact. 
-                    You can choose your own working conditions and we give our teams a lot of space. 
-                    We trust you know best and are not afraid to let you try something new! 
-                    This is how we pioneer and create more impact every day.
-                </p>
-              </blockquote>
+
+    <div class="uk-card uk-grid-collapse uk-child-width-1-2@s uk-margin uk-grid-match uk-margin-remove uk-grid">
```

Comment:
I don't think these extra indentation is necessary since it's the first line. Everything can prob be moved a bit to the left.

Reply:
Thank you @raguiar9080 . Will do.